src/App.js

```diff
@@ -2,19 +2,81 @@ import React from "react";
 import TodoList from "./TodoList";
 import AddTodoForm from "./AddTodoForm";
 
-function useSemiPersistentState() {
-  const [todoList, setTodoList] = React.useState(
-    JSON.parse(localStorage.getItem("savedTodoList")) || []
-  );
+function App() {
+  const [isLoading, setIsLoading] = React.useState(true);
+
+  const [todoList, setTodoList] = React.useState([]);
+
+  const fetchData = async () => {
+    const options = {
+      method: "GET",
+      headers: {
+        Authorization: `Bearer ${process.env.REACT_APP_AIRTABLE_API_TOKEN}`,
+      },
+    };
+    const url = `https://api.airtable.com/v0/${process.env.REACT_APP_AIRTABLE_BASE_ID}/${process.env.REACT_APP_TABLE_NAME}/`;
+
+    try {
+      const response = await fetch(url, options);
+      if (!response.ok) {
+        const message = `Error has ocurred:${response.status}`;
+        throw new Error(message);
+      }
+      const data = await response.json();
+      console.log(data);
```

Comment:
Remember to remove `console.log` from your final solution unless they are part of what the assignment is asking you. This is good practice, we never leave console.logs in our production code.